code-challenges.test.js

```diff
@@ -47,6 +68,38 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+describe("capitalLetters") = () => {
+    it("return strings with the first letter capitalized", () => {
+        expect(capitalLetters(randomNouns1)).toEqual("Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew")
+        expect(capitalLetters(randomNouns2)).toEqual("Temperature", "Database", "Chopsticks", "Mango")
+    })
+
+}
+
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
\ No newline at end of file
+
+// Pseudo code:
+// Input: The arrays that have preselected strings
+// Output: The arrays that have the first letters added to the 
+// Create a function that capitalizes the first letter in each index in the array
+// Return an output that capitalizes ONLY the first letter and not the entire word
+
+
+const capitalLetters = (array) => {
+    return array.map((value) => {
+      if (typeof value === 'string' || value instanceof String) {
+        return value.charAt(0).toUpperCase() + value.slice(1)
+      } else {
+        return value
+      }
+    })
+}
```

Comment:
I wonder if the additional edge case checks on line 91 are interfering with your code. Line 92 would take care of the necessary logic.